Assets/Mirror/Runtime/Messages.cs

```diff
@@ -153,11 +154,25 @@ public class DisconnectMessage : EmptyMessage {}
 
     public class ConnectMessage : EmptyMessage {}
 
-    public class SceneMessage : StringMessage
+    public class SceneMessage : MessageBase
     {
-        public SceneMessage(string value) : base(value) {}
+        public string sceneName;
+        public LoadSceneMode sceneMode; // Single = 0, Additive = 1
+        public LocalPhysicsMode physicsMode; // None = 0, Physics3D = 1, Physics2D = 2
```

Comment:
what is this for?

Reply:
It adds two enums to control how server and clients handle the scene message, so either or both might load the scene additively and can isolate an instance of an additive scene as a physics scene.  If the server loads umpteen instances of a single subscene, each as their own physics scene, the server can then put players in specific instances, so they play isolated from others.  A single server process could host dozens of small matches this way.

Reply:
Allows the server to tell the client the scene, scenemode and phsycics type in the scene message. So then the msg handling that already exists can add support for those things. If they are not specified in the msg then nothing different happens. Just a simple scene change

Reply:
I still don't understand what the physics mode is for. Why do we need that?

Reply:
We're all in Discord - let's discuss there.